Tests/libhostmgrTests/CommandPolicyTests.swift

```diff
@@ -0,0 +1,105 @@
+import XCTest
+@testable import libhostmgr
+
+struct TestCommand: FollowsCommandPolicies {
+    static var commandIdentifier = "TestCommandIdentifier"
+    static var commandPolicies: [CommandPolicy] = []
+}
+
+class CommandPolicyTests: XCTestCase {
+
+    private let stateStorageManager = InMemoryStorageManager()
+    private let testKey = "foo"
+
+    override func setUpWithError() throws {
+
+    }
+
+    override func tearDownWithError() throws {
+        try stateStorageManager.deleteAll()
+    }
+
+    // MARK: Evaluation
+    func testThatEvaluationIsPerformedWhenPredicateIsFalseForUnlessStatement() throws {
+        var value = false
+        to({ value = true }(), unless: false)
```

Comment:
Could you explain this syntax? I don't think I'm familiar with it.

Reply:
Sure – it's just a closure on line 25. The `to` helper method is just a bit of syntactic sugar around:

```swift
if predicate {
  expression()
}
``` 

so this test could be rewritten as:

```swift
if false {
  value = true
}
```

The test is a little silly, but the underlying helper is nice to be able to do stuff like:

```swift
try to(doStuff(), unless: currentUserIsAdmin)
```